### PR TITLE
Remove bors.toml

### PR DESCRIPTION
**Pull request by BD103**
I believe Bors is no longer being used for the website (and [all of Bevy](https://bevyengine.org/news/bevy-0-11/#no-more-bors)). This PR deletes a leftover `bors.toml` file.

Additionally, there appear to be a few other references to Bors in the codebase. Do you want me to remove them too?

- [`github_client::get_merged_prs`](https://github.com/bevyengine/bevy-website/blob/0a3c92f6163797661d28a442b7200d473e7db73d/generate-release/src/github_client.rs#L165-L168)
- [`github_client::get_merged_prs_by_page`](https://github.com/bevyengine/bevy-website/blob/0a3c92f6163797661d28a442b7200d473e7db73d/generate-release/src/github_client.rs#L227-L233)
- [`helpers::get_pr_title_from_commit`](https://github.com/bevyengine/bevy-website/blob/0a3c92f6163797661d28a442b7200d473e7db73d/generate-release/src/helpers.rs#L58-L64)
- [`main::Args`](https://github.com/bevyengine/bevy-website/blob/0a3c92f6163797661d28a442b7200d473e7db73d/generate-release/src/main.rs#L18-L28)
- [`release_notes_website::generate_release_notes_website`](https://github.com/bevyengine/bevy-website/blob/0a3c92f6163797661d28a442b7200d473e7db73d/generate-release/src/release_notes_website.rs#L39-L44)
- [`release_notes::generate_release_note`](https://github.com/bevyengine/bevy-website/blob/0a3c92f6163797661d28a442b7200d473e7db73d/generate-release/src/release_notes.rs#L39-L44)

### CODE REVIEW

**Comment by mockersf** [comment]
> Additionally, there appear to be a few other references to Bors in the codebase. Do you want me to remove them too?

Yes please!

**Review by alice-i-cecile — APPROVED** [review]

**Comment by alice-i-cecile** [comment]
Bit of a noisy diff, but I'm fine to eat that here. I spun out an issue: we can run a quick cargo fmt check in CI on this repo too.

File: generate-release/src/migration_guide.rs [hunk 1]
```diff
@@ -1,111 +1,110 @@
...
+        }
+    }
+    writeln!(&mut output, "</div>")?;
+
+    println!("\nFound {} breaking PRs merged", count);
+
+    std::fs::write(path, output)?;
+
+    Ok(())
+}
```
Comment (TrialDragon) [inline generate-release/src/migration_guide.rs:110]: Should be done in #935 instead.

**Review by TrialDragon — APPROVED** [review]
The formatting of the `helpers.rs` and `migration_guide.rs` should be kept to #935 since that is the more appropriate PR that already does this.

Otherwise LGTM.

**Comment by BD103** [comment]
> The formatting of the `helpers.rs` and `migration_guide.rs` should be kept to #935 since that is the more appropriate PR that already does this.

I agree. This PR was originally made before #935, so I assumed it would be the only thing reformatting the files. I've removed the formatting, so this should be ready to merge.

**Review by mockersf — APPROVED** [review]

**Comment by BD103** [comment]
After further research, the reason the diff was so messy was due to inconsistent line endings. I created a separate PR (#939) that should hopefully fix that.

**Pull request merged**